src/node/src/client.js

```diff
@@ -184,14 +184,15 @@ function _emitStatusIfDone() {
     } else {
       status = this.received_status;
     }
-    this.emit('status', status);
-    if (status.code !== grpc.status.OK) {
+    if (status.code === grpc.status.OK) {
+      this.push(null);
```

Comment:
Yes. Pushing `null` signifies the end of the stream, which causes the `end` event to be emitted.